rfc6962-bis.xml

```diff
@@ -465,6 +465,9 @@ REDACT(label) = prefix || BASE32(index || LABELHASH(keyid || label))</artwork>
           Logs MUST verify that each submitted certificate or precertificate has a valid signature chain to an accepted trust anchor, using the chain of intermediate CA certificates provided by the submitter. Logs MUST accept certificates and precertificates that are fully valid according to <xref target="RFC5280">RFC 5280</xref> verification rules and are submitted with such a chain. Logs MAY accept certificates and precertificates that have expired, are not yet valid, have been revoked, or are otherwise not fully valid according to RFC 5280 verification rules in order to accommodate quirks of CA certificate-issuing software. However, logs MUST reject submissions without a valid signature chain to an accepted trust anchor. Logs MUST also reject precertificates that do not conform to the requirements in <xref target="Precertificates"/>.
         </t>
         <t>
+          Logs MUST reject submission of precertificates where both the subjectAltName and <xref target="redacted_san_extension">redactedSubjectAltName</xref> extensions are present.
```

Comment:
This added paragraph seems redundant.

The previous paragraph already says "Logs MUST also reject precertificates that do not conform to the requirements in Section 3.2", and Section 3.2 already says "the subjectAltName [RFC5280] extension MUST be omitted when the CA is using domain name redaction (Section 4.2)."

Instead of adding this paragraph, how about rewording the requirement in Section 3.2 to something programmatically enforceable?  e.g.
s/when the CA is using domain name redaction/when the redactedSubjectAltName extension is present/